### DISCUSSION TITLE
Disable / Allow permission for use enchantment table

### DISCUSSION THREAD

**Discussion by MykeUhu** [EssentialsX/Essentials#4495] [Q&A] [answered]
![Bild_2021-08-29_145002](https://user-images.githubusercontent.com/50227900/131250909-9b8f9ad6-8ca4-40a8-9718-13acb24e8b0d.png)
Is there a way to allow / disable the Enchantment table?
 modifyworld.items.enchant.* false
would not work..
greetings

**Comment by Chew** [comment, marked as answer]
You don't have EssentialsX NOR EssentialsXAntiBuild installed, both of which are required

Additionally, modifyworld isn't even an Essentials permission node 🤔 
You'd probably want to negate `essentials.build.interact.enchantment_table` (or something)
